snap/snapcraft.yaml

```diff
@@ -45,11 +43,13 @@ parts:
       - pkg-config
       - rsync
       - tcl
+      - sudo
```

Comment:
why do we need sudo ?

Reply:
The new build bases (that comes with the craft-application update) does not have sudo installed by default and our Makefile(via build scripts) call it